### ISSUE TITLE
[Token Management] Separate single/grouped sort position

### ISSUE THREAD

**Issue by caybro** [status-im/status-desktop#13276]
# Bug Report
We've just been saving the `customSortOrderNo` without the distinction whether it's a flat position (for single tokens) or a grouped position. Need to add an extra field to the settings data:
```cpp
// symbol -> {sortOrder, visible, groupId}
using SerializedTokenData = QHash<QString, std::tuple<int, bool, QString>>;
```

**Comment by caybro** [comment]
Unlikely to be finished in time for this milestone... needs substantial amount of work, move to next milestone?

**Comment by noeliaSD** [comment]
It's not a refactor, now the groups arrangement are not working fine. Am I right @caybro . I think it's a high priority task since the `Tokens Management` functionality is not finished bc of this!

cc: @iurimatias @alexandraB99

**Comment by caybro** [comment]
> It's not a refactor itself, now the groups arrangement are not working fine and this change is needed to make it properly work. Am I right @caybro ? I think it's a high priority task since the `Tokens Management` functionality is not finished bc of this!
> 
> cc: @iurimatias @alexandraB99

Yup agreed, scheduled for one of the upcoming sprints for this milestone
